Structure_Engine/Query/AnalyticalBars.cs

```diff
@@ -45,9 +46,10 @@ public static partial class Query
 
```

Comment:
Depends how you look at it I think. The Mass is not a property of the bar either, but still available through a query method. Could, and have been, arguing that this would be an more extreme case of that. "Get the analytical bars from the physical framing". At the other end we have a Create method for framingElement that accepts a bar. 

But, I do get your point.